demos/twa-play-billing/src/main/AndroidManifest.xml

```diff
@@ -67,6 +67,7 @@
         <activity
             android:name="com.google.androidbrowserhelper.playbilling.provider.PaymentActivity"
             android:theme="@android:style/Theme.Translucent.NoTitleBar"
+            android:configChanges="keyboardHidden|orientation|screenSize"
```

Comment:
I'm not super familiar with these, but looking at the documentation (https://developer.android.com/guide/topics/manifest/activity-element#config) should this also include "keyboard" and "screenLayout"?

Reply:
Sure, done.